sample-apps/liberty-app/build.gradle

```diff
@@ -5,13 +5,15 @@ plugins {
 dependencies {
   providedCompile 'javax:javaee-api:8.0.1'
   providedCompile 'org.eclipse.microprofile:microprofile:3.0'
-  testCompile project(':microshed-testing-liberty')
-  testCompile project(':microshed-testing-core')
-  testCompile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.36'
+  testImplementation 'org.testcontainers:junit-jupiter:1.19.1'
+  testImplementation project(':microshed-testing-liberty')
+  testImplementation project(':microshed-testing-testcontainers')
```

Comment:
Liberty includes `microshed-testing-testcontainers` no need to include it here.
```suggestion
```

Reply:
I thought so too but since change some gradle dependencies from compile to implementation, the behavior is different. That's why I need your help, I'm not gr8 at gradle. Without this explicit dependency in this sample app, it will not compile. This leads me to believe that the dependency: testcontainers is no longer part of the artifact created in the liberty module.